docs/DevelopersGuide.adoc

```diff
@@ -1478,626 +2262,534 @@ include::../src-docs/com/wsscode/pathom/book/async/js_promises.cljs[]
+benefits: Tools give better support (e.g. query autocompletion within Fulcro Inspect), and you can
+add your own client-side resolvers that can derive new shapes/data for the API, making it possible
+to shape the external API to your local UI whims.
 
-Let's write some boilerplate to kickstart the project:
+In both cases Pathom includes implementations of Fulcro Remotes, so you can easily drop GraphQL
+support into a Fulcro application as a remote!
 
-```clojure
-(ns pathom-docs.connect.getting-started
-  (:require [com.wsscode.pathom.core :as p]
-            [com.wsscode.pathom.connect :as p.connect]))
+This chapter assumes you're familiar with Pathom's <<AsyncParsing, async support>>.
 
-(def parser
-  (p/parser {::p/plugins
-             [(p/env-plugin
-                {::p/reader [p/map-reader
-; to use connect with the async parser, use the p.connect/all-async-readers instead
-                             p.connect/all-readers]})]}))
+The namespaces concerned are:
 
-(comment
-  (parser {::p/entity {:hello "World"}} [:hello]))
+```
+    [com.wsscode.pathom.graphql :as pg]
+    [com.wsscode.pathom.connect.graphql :as pcg]
+    [com.wsscode.pathom.fulcro.network :as pfn]
 ```
 
-`Connect` reader is used in conjunction with the <<map-reader,map reader>>, when the entity doesn't have the information, `Connect` will be triggered to resolve the attribute.
-
-=== Global resolvers
-
-To start simple, let's create an entry point that provides the latest product we have in our store, to accomplish that we need to write a `resolver`, create an `index` and then use that to run our query:
-
-[source,clojure]
-----
-include::../src-docs/com/wsscode/pathom/book/connect/getting_started.cljs[]
-----
-
-We have some rules for the `resolver` functions:
-
-. It always takes two arguments:
-.. the environment, which is provided by the regular parser engine
-.. a map containing the required input data for that `resolver` (more on this later).
-. It must return a map, with at least one key.
-
-The critical thing to notice here is: resolvers always take named parameters (input map) and always spit named attributes (output map). This structure enables for automatic attribute walking, which we will see later in this tutorial.
+== Simple GraphQL
 
-In our first resolver we expose the attribute `::latest-product`, and this resolver doesn't require any input, from now one we will call those `global resolvers` (those which don't require any input, so can be requested anywhere). Also, note that in our output description we have the full output details (including nested attributes), this is mostly useful for auto-complete on UI's and automatic testing.
+The main functions for dealing with translating to/from GraphQL are:
 
-Try some of these queries on the demo below:
+`pg/query->graphql` - Converts and EDN graph query to a GraphQL string.
+`pfn/graphql-response-parser` - A parser that can take a GraphQL response and an EDN query to generate a proper response
```

Comment:
not sure if this is worth mentioning, it's more an internal parser to fetch back the data. what you think?

Reply:
Agreed.  I wrote that before we talked, and thought things on the "basic" end were more low-level.  Now I see that you probably would never use those directly.